4-3-5.py

```diff
@@ -0,0 +1,46 @@
+'''
+На вход программе подается строка текста, содержащая символы. Напишите программу, которая упаковывает последовательности одинаковых символов заданной строки в подсписки.
+
+Формат входных данных
+На вход программе подается строка текста, содержащая символы, отделенные символом пробела.
+
+Формат выходных данных
+Программа должна вывести указанный вложенный список
+
+Sample Input 1:
+
+a b c d
+
+Sample Output 1:
+
+[['a'], ['b'], ['c'], ['d']]
+
+Sample Input 2:
+
+w w w o r l d g g g g r e a t t e c c h e m g g p w w
+
+Sample Output 2:
+
+[['w', 'w', 'w'], ['o'], ['r'], ['l'], ['d'], ['g', 'g', 'g', 'g'], ['r'], ['e'], ['a'], ['t', 't'], ['e'], ['c', 'c'], ['h'], ['e'], ['m'], ['g', 'g'], ['p'], ['w', 'w']]
+'''
+
+def groupizm(s):
+    in_list = []
+    f_list = []
+    res = []
+    for _ in s:
```

Comment:
`_` is usually used for variables that are not used. You use it below

Reply:
Complit! Thx master!